content/blog/2016/2016-09-28-jenkins-world-2016-wrap-up-hub.adoc

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: "Jenkins World 2016 Wrap-up - Open Source Hub"
```

Comment:
I suppose more clearly, I'm not sure people who weren't at the event would recognize "Open Source Hub" verbiage, whereas anybody (almost) who has ever attended a JUC or Jenkins World should probably know what "Ask the Experts" entails